public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -67,37 +70,140 @@ a#i18n-attribute
     It will be removed by the compiler _after_ translation.
 
 :marked
-  In the accompanying sample, an `<h1>` tag displays a simple English language greeting which you will translate to French:
+  In the accompanying sample, an `<h1>` tag displays a simple English language greeting which you will translate to Spanish:
 +makeExample('cb-i18n/ts/app/app.component.1.html', 'greeting', 'app/app.component.html')(format=".")
 :marked
   Add the `i18n` attribute to the tag to mark it for translation.
 
 +makeExample('cb-i18n/ts/app/app.component.1.html', 'i18n-attribute', 'app/app.component.html')(format=".")
 
 :marked
+  ### Help the translator with a _description_ and _intent_
+
   The translator may need a description of the message to translate it accurately.
   Assign a description to the i18n attribute:
   
 +makeExample('cb-i18n/ts/app/app.component.1.html', 'i18n-attribute-desc', 'app/app.component.html')(format=".")
 
 :marked
-  The true _meaning_ of the text may require some application context.
-  Add a contextual meaning to the assigned string, separating it from the description with the `|` character:
+  The translator may need to know your _intent_ &mdash; 
+  the true _meaning_ of the text within _this particular_ application context &mdash; 
+  in order to deliver a correct translation.
+  Add some contextual meaning to the assigned string, in front of the description,
+  separating it from the description with the `|` character (`<meaning>|<description>`):
   
 +makeExample('cb-i18n/ts/app/app.component.html', 'i18n-attribute-meaning', 'app/app.component.html')(format=".")
 
 :marked
-  While all appearance of a message with the _same_ meaning should have the _same_ translation, 
+  While all appearances of a message with the _same_ meaning have the _same_ translation, 
   a message with *different meanings* could have different translations.
   The Angular extraction tool preserves both the _meaning_ and the _description_ in the translation source file
   to facilitiate contextually-specific translations.
 
+  ### Translate text without creating an element
+
+  Suppose there is a stretch of text that you'd like to translate.
```

Comment:
Yes.  I think we don't have to explain all the reasons you'd do this. The CSS one was compelling enough.